canonical-web-and-design/canonical.com
# Update partner programmes

## Done

- Finishes work started in #221
- Includes a fair bit of work on the partners db
- Update secondary nav spacing
- Still might require work on become-a-partner

## QA

- Check out this feature branch
- Run the site using the command `dotrun serve`
- Visit http://0.0.0.0:8002/ and click on Partners in the navigation
Navigate to http://0.0.0.0:8002/partners and see the available partner programmes. QA the links on this page
QA all the links on the following pages: http://0.0.0.0:8002/partners/public-cloud, http://0.0.0.0:8002/partners/ihv-and-oem, http://0.0.0.0:8002/partners/desktop, http://0.0.0.0:8002/partners/channel-and-reseller, http://0.0.0.0:8002/partners/gsi & http://0.0.0.0:8002/partners/become-a-partner


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2559 & Fixes #214
